examples/validate_relative_binding.py

```diff
@@ -22,27 +22,25 @@
 from fe.utils import convert_uM_to_kJ_per_mole
 from fe import model_rabfe
 from fe.free_energy_rabfe import RABFEResult
+from fe.atom_mapping import CompareDistNonterminal
 
 from ff import Forcefield
 from ff.handlers.deserialize import deserialize_handlers
 from ff.handlers.serialize import serialize_handlers
 from parallel.client import CUDAPoolClient, GRPCClient
 from parallel.utils import get_gpu_count
 
-import multiprocessing
+
 from training.dataset import Dataset
 from rdkit import Chem
 
 import timemachine
 from timemachine.potentials import rmsd
 from md import builders, minimizer
 from rdkit.Chem import rdFMCS
-from fe.atom_mapping import CompareDistNonterminal
 
 if __name__ == "__main__":
 
-    multiprocessing.set_start_method('spawn')
```

Comment:
@proteneer Based on my understanding of multiprocessing, we can rely on the `get_context` in the `CUDAPoolClient`. If this was added for another use case let me know

Reply:
IIRC this method must be called only once, and at the start of the run. This was added at the start to basically detect if anything else (imports etc.) either called set_start_method before (in which this line would fail), or after (in which the later line would fail).